angularSTEP/src/app/discover-page/discover-page.component.html

```diff
@@ -74,11 +74,10 @@
         <app-search-box (noQuery)='displayResults($event)' contentType='Search for users' [delay]='500'></app-search-box>
           <ais-hits *ngIf='showResults'>
             <ng-template let-hits="hits">
-                <div class="search-content-container">
-                  <app-profile-card *ngFor="let user of hits" (click)='goToUser(user.username)'
-                    [user]='{displayName: user.displayName, username: user.username, picUrl: user.picUrl}'>
-                  </app-profile-card>
-                </div>
+              <div class="search-content-container">
+                <app-profile-card *ngFor="let user of hits" [user]='user'>
```

Comment:
Can you no longer click to go to the users page?

Reply:
Good catch, however, you can, it just is in a previous PR that will get merged to main before this one.

Reply:
Cool just making sure

Reply:
Could you please reference the specific PR that needs to be merged before this one?

Reply:
I guess it would have been good for me to check. I thought it was #88 however I went to look and its not? I will add it to this PR.